jetty/src/main/scala/org/http4s/server/jetty/JettyConfig.scala

```diff
@@ -0,0 +1,214 @@
+package org.http4s
+package server
+package jetty
+
+import java.net.InetSocketAddress
+import java.util
+import java.util.{EnumSet, UUID}
+import java.util.concurrent.Executor
+import javax.net.ssl.SSLContext
+import javax.servlet.{DispatcherType, Filter}
+import javax.servlet.http.HttpServlet
+
+import org.eclipse.jetty.server.{Server => JServer, _}
+import org.eclipse.jetty.servlet.{FilterHolder, ServletHolder, ServletContextHandler}
+import org.eclipse.jetty.util.Jetty.{VERSION => JettyVersion}
+import org.eclipse.jetty.util.component.AbstractLifeCycle.AbstractLifeCycleListener
+import org.eclipse.jetty.util.component.LifeCycle
+import org.eclipse.jetty.util.ssl.SslContextFactory
+import org.http4s.internal.compatibility._
+import org.http4s.internal.kestrel._
+import org.http4s.servlet.{ServletContainer, Http4sServlet, Http4sServletConfig}
+import org.http4s.tls.ClientAuth
+import org.log4s.getLogger
+import scalaz.concurrent.Task
+
+/** Configures an embedded Jetty server. */
+private[jetty] abstract class JettyConfigBase { self: JettyConfig =>
+  private[this] val log = getLogger
+
+  /** Returns a task to start a Jetty server.  Call one of the
+   * `.unsafePerform` methods on the task to start the server.
+   *
+   * @param jetty A Jetty server instance to be configured
+   */
+  def start: Task[Server] =
+    Task.delay {
+      val jetty = new JServer(threadPool.orNull)
+      configureJetty(jetty)
+      jetty.start()
+      banner.foreach(_.lines.foreach(log.info(_)))
+      log.info(s"Started http4s-${BuildInfo.version} on jetty-${JettyVersion}")
+
+      new Server {
+        val address =
+          jetty.getConnectors.collectFirst {
+            case connector: NetworkConnector =>
+              val host = Option(connector.getHost).getOrElse("0.0.0.0")
+              val port = connector.getLocalPort
+              new InetSocketAddress(host, port)
+          }.getOrElse(new InetSocketAddress("0.0.0.0", 0))
```

Comment:
Was `0.0.0.0` the default before?  Remember loopback versus all interfaces being discussed before, but don't recall the outcome.